cfgov/v1/jinja2/v1/layouts/base.html

```diff
@@ -21,7 +21,10 @@
     ============================================================================
 -->
 {% endif %}
-<html lang="{{ language }}" class="no-js">
+<html lang="{{ language }}" class="no-js"
```

Comment:
hmmm, I think we might need to massage the templates a bit to make this align with how the schema is supposed to work, namely, the itemscope of the page probably should be on `main` since that's where the actual QA material is.

So I think this might look more like folding the approach taken in `content-base` directly into `base.html` to maintain the attributes on the `main` tag.

Reply:
I was going off the FAQPage example Google provides on https://developers.google.com/search/docs/appearance/structured-data/faqpage, which places it on `html` (I guess because FAQPage is one of the types of https://schema.org/WebPage, so the whole page is the context). 

However, Google's same documentation has QAPage—which should be at the same hierarchy as FAQPage relative to WebPage—applied to the `body`, see https://developers.google.com/search/docs/appearance/structured-data/qapage. 

Reading further I noticed there is a guideline:

> Don't use QAPage markup for content that has only one answer for a given question with no way for users to add alternative answers; instead, use FAQPage.

So, I think I misinterpreted the difference between these two and I should switch it back to FAQPage! Reverted back in https://github.com/cfpb/consumerfinance.gov/pull/8076/commits/e0cc96a7868daab7f58c9adf40109a1ed9d02ca9

With the Google docs in mind, do you think it's still problematic to have it on `html`? I'm not immediately seeing explicit guidance on where to place it. It's true if there were another `itemscope`, it would be inside the FAQPage, but I'm not sure that would matter. What does moving it to `main` accomplish?

Reply:
I was keying off [section 1b](https://schema.org/docs/gs.html) where it implies you should be scoping things pretty fine-grained... but if google doesn't care then I don't think we do either.